frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import OperatorBackedOwnerReferences from '@console/internal/components/utils';
+import { TopologyDataObject } from '../topology-types';
+import TopologyHelmReleaseResourcesPanel from '../helm/TopologyHelmReleaseResourcesPanel';
+import { OverviewItem } from '@console/shared';
+
+type TopologyOperatorBackedResourcesProps = {
+  item: TopologyDataObject;
+};
+
+const TopologyOperatorBackedResources: React.FC<TopologyOperatorBackedResourcesProps> = ({
+  item,
+}) => {
+  const { groupResources = [] } = item;
+  const finalRes = groupResources.map((val) => val.resources.obj);
+  const ownerReferencedResources = groupResources.reduce(
+    (acc, val) => {
+      acc.obj = { ...val.resources.obj, ...acc.obj };
+      return acc;
+    },
+    { obj: {}, isOperatorBackedService: true },
+  );
+
+  return (
+    <div className="overview__sidebar-pane-body">
+      <OperatorBackedOwnerReferences item={ownerReferencedResources as OverviewItem} />
+      <TopologyHelmReleaseResourcesPanel
+        manifestResources={finalRes}
+        releaseNamespace={item.resources.obj.metadata.namespace}
+      />
```

Comment:
Probably better to rename this component to something generic that both operator and helm side panels use. Something like `TopologyWorkloadResourcesPanel`. WDYT @jeff-phillips-18 ?

Reply:
agreed.